Tests/js/views/assets/ez-usermenuview-tests.js

```diff
@@ -0,0 +1,157 @@
+            delete this.view;
+        },
+
+        "Should add item to the user menu items": function () {
+            this.view.addMenuItem(this.userMenuItemSecondView);
+
+            Y.Assert.areSame(
+                2,
+                this.view.get('items').length,
+                'Should add menu item to the items attribute'
+            );
+        },
+    });
+
+    attrTest = new Y.Test.Case({
+        name: "eZ User Menu attribute test",
+
+        setUp: function () {
+            Y.eZ.UserMenuItemFireEventView = Y.View;
+            this.view = new Y.eZ.UserMenuView();
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should pass the attribute to the view": function () {
+            var item = this.view.get('items')[0];
+
+            Y.Assert.areSame('logOut', item.get('eventName'), 'Should pass the correct event name');
+            Y.Assert.areSame('Logout', item.get('title'), 'Should pass the correct title');
+            Y.Assert.areSame(this.view, item.getTargets()[0], 'Should set correct event target');
+        },
+    });
+
+    displayTest = new Y.Test.Case({
+        name: "eZ User Menu display test",
+
+        setUp: function () {
+            this.userMenuItemFireEventView = new Y.View();
+            this.view = new Y.eZ.UserMenuView({
+                items: [this.userMenuItemFireEventView],
+                container: '.container'
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should show the user menu": function () {
+            var view = this.view,
+                container = view.render().get('container');
+
+            view.toggleDisplayed();
+
+            Y.Assert.isFalse(container.hasClass(CLASS_HIDDEN), 'The user menu should be visible');
+            Y.Assert.isTrue(view.get('displayed'), 'Value of displayed attribute should be true');
+        },
+
+        "Should hide the user menu": function () {
+            var view = this.view,
+                container = view.render().get('container');
+
+            view.toggleDisplayed();
+            view.toggleDisplayed();
+ 
+            Y.Assert.isTrue(container.hasClass(CLASS_HIDDEN), 'The user menu should be hidden');
+            Y.Assert.isFalse(view.get('displayed'), 'Value of displayed attribute should be false');
+        },
+
+        "Should hide the user menu after clicked outside": function () {
+            var view = this.view.render();
+
+            view.toggleDisplayed();
+
+            view.after('displayedChange', function () {
+                Y.Assert.isTrue(view.get('container').hasClass(CLASS_HIDDEN), 'The user menu should be hidden');
+                Y.Assert.isFalse(view.get('displayed'), 'Value of displayed attribute should be false');
+            });
+
+            Y.one(SELECTOR_OUTSIDE).simulate('click');
```

Comment:
the `displayedChange` event handler should set a boolean flag so that you verify that it was actually called. Otherwise, it can not be called and the test will still pass.